docker/README.md

```diff
@@ -75,3 +75,19 @@ Enter docker container to check any issues
 ```
 docker exec -it <container_id|container_name> /bin/bash
 ```
+
+How to Collect Docker Container Performance Metrics using OMI/OMI provider in Docker Container
+-------
+
+OMI/OMI provider can work on containers, just make sure you know following notes:
+- OMI/OMI provider can collect containers performance metrics in a normal container, but you cannot collect virtual machine host's performance metrics in container.
```

Comment:
After investigation, OMS doesn't not install in containers but in host, and it collect all perf from host, even OMS can run inside containers.

I see "Installing and configuring the solution" as following :https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-containers
```
There are a few ways to install and use Docker with OMS:+
On supported Linux operating systems, install and run Docker and then install and configure the OMS Agent for Linux.
On CoreOS, you cannot run the OMS Agent for Linux. Instead, you run a containerized version of the OMS Agent for Linux.
On Windows Server 2016 and Windows 10, install the Docker Engine and client then connect an agent to gather information and send it to Log Analytics.
```